DQM/BeamMonitor/plugins/BeamSpotDipServer.cc

```diff
@@ -57,7 +57,7 @@ BeamSpotDipServer::BeamSpotDipServer(const edm::ParameterSet& ps) {
   dip = Dip::create("CmsBeamSpotServer");
 
   //
-  dip->setDNSNode("dipns1,dipns2");
+  dip->setDNSNode("cmsdimns1.cern.ch");
```

Comment:
why not configure it via parameter set?

Reply:
Hi @smuzaffar! There are a few other suggestions that you gave in the past on the DIP client and that I have on my todo list for future improvements of this plugin. Is it ok if I add this one to the list and we take care of it in the future?

Reply:
sure, it is ok.